server/src/Core/Domain/Common/Model/VO/ResourceUuid.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the G.L.S.R. Apps package.
+ *
+ * (c) Dev-Int Création <info@developpement-interessant.com>.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Core\Domain\Common\Model\VO;
+
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+class ResourceUuid implements ResourceUuidInterface
```

Comment:
Did you simplify the Uuid management mechanism?

Reply:
yes